x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/cluster_details_box.tsx

```diff
@@ -64,8 +65,8 @@ export const ClusterDetailsBox = ({ cluster }: { cluster: Cluster }) => {
           }
         >
           <EuiLink onClick={() => handleClusterTitleClick(cluster.meta.clusterId)} color="text">
-            <EuiText size="xs">
-              <h3>
+            <EuiTitle css={{ fontSize: 20 }}>
```

Comment:
`20` is the only size that doesnt make the UI looks bad here. but euiTheme sizes are either 16 or 24.
Settings custom font sizes is fairly common in kibana i wouldn't worry about that one.